### ISSUE TITLE
SheetViewport isn't importable in the latest version 0.10.2

### ISSUE THREAD

**Issue by andcea** [fujidaiti/smooth_sheets#291]
I noticed `SheetViewport` was made public in recent commits however, those haven't been published as a new version yet.

When following the examples (for example [declarative_navigation_sheet.dart](https://github.com/fujidaiti/smooth_sheets/blob/main/example/lib/tutorial/declarative_navigation_sheet.dart ) or [ai_playlist_generator.dart](https://github.com/fujidaiti/smooth_sheets/blob/main/example/lib/showcase/ai_playlist_generator.dart)) `SheetViewport` is used and it won't work with the latest version of the library.

You can get it working by changing your pubspec.yaml to:
```yaml
  smooth_sheets:
    git:
      url: https://github.com/fujidaiti/smooth_sheets.git
      ref: 5fad0b8bab04fe88cc08d5c43e9d7edd1efa95be
```

but publishing a new library version would be great.

**Comment by fujidaiti** [comment]
Hi @andcea,

Sorry for the confusion.  
It is part of the new features that will be included in version 0.11.0, which is scheduled to be released by the end of January. In the meantime, please refer to the examples in [this branch](https://github.com/fujidaiti/smooth_sheets/tree/v0.x).

**Comment by andcea** [comment]
Thanks @fujidaiti! I'll close this as it sounds like it will be part of the next release.
